mypy.ini

```diff
@@ -44,9 +44,6 @@ ignore_errors = True
 [mypy-keylime.revocation_notifier]
 ignore_errors = True
 
-[mypy-keylime.tenant]
-ignore_errors = True
-
```

Comment:
The registrar_client.py can also be removed. It probably got lost during rebasing.